### PR TITLE
Add another paragraph about mentioning another needed library

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: doc/filters/markdown_to_html.rst [hunk 1]
```diff
@@ -60,3 +62,6 @@ You can also use the filter on an included file:
                 }
             }
         });
+       
+    Afterwards you need to install a markdown library of your choice. Some of them are
+    mentioned in ``require-dev`` section of ``twig/markdown-extra``.
```
Comment (xabbuh) [inline doc/filters/markdown_to_html.rst]:
```rst
[...] in the ``require-dev`` section of the ``twig/markdown-extra`` package.
```

Comment (OskarStark) [reply doc/filters/markdown_to_html.rst]: Done

**Comment by OskarStark** [comment]
Not sure this is true for `2.x` 🤔

**Comment by fabpot** [comment]
Same is true for 2.x

**Comment by OskarStark** [comment]
> Same is true for 2.x

@fabpot can you switch while merging or shall I rebase against `2.x` ?

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**